nipype/pipeline/engine/nodes.py

```diff
@@ -619,7 +618,6 @@ def _run_command(self, execute, copyfiles=True):
             inputs=self._interface.inputs.get_traitsfree())
 
         if copyfiles:
-            self._originputs = deepcopy(self._interface.inputs)
```

Comment:
is this no longer being used in report.rst ?

Reply:
```Shell
$ grep -rI _originputs nipype
nipype/pipeline/engine/nodes.py:        self._originputs = None
nipype/pipeline/engine/nodes.py:            self._originputs = deepcopy(self._interface.inputs)
```

Reply:
ah yes - this was changed in oscar's node refactor.